### PR TITLE
Add 26 Jan Seminar

### PR DESCRIPTION
**Pull request by danforbes**
@jimmychu0807 please do not merge until we get approval from @dastanbeksamatov.

cc @EmilyOstbo

### CODE REVIEW

**Review by jimmychu0807 — APPROVED** [review]
lgtm

**Comment by jimmychu0807** [comment]
@danforbes So I will just approve first, you confirm the date and merge it.

File: website/pages/en/seminar.js [hunk 1]
```diff
@@ -94,6 +94,8 @@ function Seminar(props) {
               <h4 className='mb-2'><translate>What topics are coming up?</translate></h4>
               <p>On <strong>12 Jan</strong> Parity's Master of Validators, Will Pankiewicz, will discuss network security.</p>
               <p>On <strong>19 Jan</strong> Substrate Core Developer Guillaume Tholliere will discuss FRAME v2.</p>
+              <p>On <strong>26 Jan</strong> Dastan Beksamatov from <a href='https://limechain.tech/'>LimeChain</a> will discuss the design&nbsp;
```
Comment (dastanbeksamatov) [inline website/pages/en/seminar.js:97]: My full name is: Dastanbek Samatov. And shortly, just Dastan)

**Comment by danforbes** [comment]
I have no idea why I closed this PR and deleted the branch 😅

File: website/pages/en/seminar.js (hunk 1 above)
Comment (danforbes) [reply website/pages/en/seminar.js]: Sorry about that - are you okay with me using "Dastan" or do you want to use your full name?

**Review by dastanbeksamatov — APPROVED** [review]

**Pull request merged**